docs/pages/deploy-a-cluster/multi-region-blueprint.mdx

```diff
@@ -0,0 +1,314 @@
+---
+title: "Multi-region blueprint"
+description: "Blueprint describing how to deploy a multi-region Teleport Enterprise cluster using CockroachDB."
+---
+
+## Introduction
+
+This page describes how to deploy a Teleport cluster in multiple regions to
+improve resiliency and sustain regional failure. This is not a step-by-step
+guide but a blueprint on how to build a multi-region Teleport cluster. This
+blueprint must be adapted to work with your specific infrastructure and network
+constraints.
+
+## Important technical consideration
+
+Before continuing, you must acknowledge the following warnings:
+
+- This setup is not required for most Teleport users and customers.
+  Single-region multi-zone deployments provide the best availability/cost ratio.
+  For example, Amazon SLAs are 99.99% for both DynamoDB and EC2.
+- This blueprint relies on many complex components. If you don't have the team
+  capacity and expertise to deploy and operate such components, you will end
+  up with a lower availability than a single-region multi-zone setup.
+- You are responsible for deploying and maintaining the Teleport dependencies
+  including the database, object storage, container runtime, network plugins, ...
+
+<Admonition type="tip">
+
+For the teams with limited bandwidth who need multi-region clusters, Teleport
+offers to host cross-region clusters. You can benefit from the
+setup described in this document without the operational costs and complexity,
+see [the Teleport Enterprise Cloud page](../choose-an-edition/teleport-cloud/introduction.mdx)
+for more details.
+
+</Admonition>
+
+## Architecture overview
+
+A multi-region Teleport deployment is composed of several regional Teleport
+deployments. Teleport stores its state in a multi-region backend and uses
+[Proxy Peering](../architecture/proxy-peering.mdx) to connect users to resources
+hosted a different region.
+
+The recommended multi-region backend
```

Comment:
Isn't this the only supported multi-region backend?

Reply:
It's complex. Firestore is multi-region already, and you can definitely achieve multi-region with etcd (althought the latency 🤢) , and with postgres if you know the DB enough.